src/content/developers/docs/programming-languages/dart/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Ethereum for Dart Developers
+description: Learn how to develop for Ethereum using the Dart language
+lang: en
+sidebar: true
+incomplete: true
+---
+
+## Getting Started with Smart Contracts and the Solidity Language {#getting-started-with-smart-contracts-and-solidity}
+
+## Tutorials {#tutorials}
+
+1. [This tutorial](https://www.geeksforgeeks.org/flutter-and-blockchain-hello-world-dapp/) takes you through all the steps to get started:
```

Comment:
Would just change the outer list here to an unordered list (bullets) since order doesn't matter. Should help with readability since there is a nested ordered list here.